.vib/solr/ginkgo/solr_test.go

```diff
@@ -0,0 +1,109 @@
+package solr_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	utils "github.com/bitnami/charts/.vib/common-tests/ginkgo-utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	PollingInterval = 1 * time.Second
+)
+
+var _ = Describe("Solr", Ordered, func() {
+	var c *kubernetes.Clientset
+	var ctx context.Context
+	var cancel context.CancelFunc
+
+	BeforeEach(func() {
+		ctx, cancel = context.WithCancel(context.Background())
+
+		conf := utils.MustBuildClusterConfig(kubeconfig)
+		c = kubernetes.NewForConfigOrDie(conf)
+	})
+
+	When("a collection is created and Solr is scaled down to 0 replicas and back up", func() {
+		It("should have access to the created collection", func() {
+
+			getAvailableReplicas := func(ss *appsv1.StatefulSet) int32 { return ss.Status.AvailableReplicas }
+			getSucceededJobs := func(j *batchv1.Job) int32 { return j.Status.Succeeded }
+			getOpts := metav1.GetOptions{}
+
+			By("checking all the replicas are available")
+			ss, err := c.AppsV1().StatefulSets(namespace).Get(ctx, stsName, getOpts)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(ss.Status.Replicas).NotTo(BeZero())
+			origReplicas := *ss.Spec.Replicas
+
+			Eventually(func() (*appsv1.StatefulSet, error) {
+				return c.AppsV1().StatefulSets(namespace).Get(ctx, stsName, getOpts)
+			}, timeout, PollingInterval).Should(WithTransform(getAvailableReplicas, Equal(origReplicas)))
+
+			svc, err := c.CoreV1().Services(namespace).Get(ctx, stsName, getOpts)
+			Expect(err).NotTo(HaveOccurred())
+
+			port, err := utils.SvcGetPortByName(svc, "tcp-client")
+			Expect(err).NotTo(HaveOccurred())
+
+			image, err := utils.StsGetContainerImageByName(ss, "solr")
+			Expect(err).NotTo(HaveOccurred())
+
+			// Use current time for allowing the test suite to repeat
+			currentTime := time.Now()
+			jobSuffix := fmt.Sprintf("%d%d%d%d%d%d",
+				currentTime.Year(), currentTime.Month(),
+				currentTime.Day(), currentTime.Hour(),
+				currentTime.Minute(), currentTime.Second())
```

Comment:
Use `time.Format()`:
```suggestion
			// Use current time for allowing the test suite to repeat
			jobSuffix := time.Now().Format("20060102150405")
```